go/registry/api/api.go

```diff
@@ -572,18 +572,16 @@ func VerifyRegisterNodeArgs( // nolint: gocyclo
 		)
 		return nil, nil, err
 	}
-	certPub, err := verifyNodeCertificate(logger, &n)
+	_, err := verifyNodeCertificate(logger, &n)
 	if err != nil {
 		return nil, nil, err
 	}
-	if !sigNode.MultiSigned.IsSignedBy(certPub) {
-		logger.Error("RegisterNode: not signed by TLS certificate key",
-			"signed_node", sigNode,
-			"node", n,
-		)
-		return nil, nil, fmt.Errorf("%w: registration not signed by TLS certificate key", ErrInvalidArgument)
+	if n.Committee.NextCertificate != nil {
+		_, grr := verifyNodeNextCertificate(logger, &n)
```

Comment:
Why a separate function, it looks similar?

Reply:
Fixed, thanks!